onnxruntime/core/providers/rocm/rocm_profiler.h

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+#include <vector>
+#include <map>
+
+#include "core/common/profiler_common.h"
+
+#if defined(USE_ROCM) && defined(ENABLE_ROCM_PROFILING)
+
+class RoctracerLogger;
+class roctracerRow;
+
+namespace onnxruntime {
+
+namespace profiling {
+
+using Events = std::vector<onnxruntime::profiling::EventRecord>;
+
+class RocmProfiler final : public EpProfiler {
+ public:
+  RocmProfiler();
+  RocmProfiler(const RocmProfiler&) = delete;
+  RocmProfiler& operator=(const RocmProfiler&) = delete;
+#if 0
```

Comment:
Need cleanup?

Reply:
Do you see any reason why the profiler need to implement move semantics like this?  Cuda_profiler does this.  I was going to just drop it.  @RandySheriffH  What is the use case for this please?